functions/.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
+  "recommendations": [
+    "ms-azuretools.vscode-azurefunctions",
+    "ms-python.python"
+  ]
+}
```

Comment:
We might need to remove all of the .vsode files from our folder. I think in toddy's lecture he recommended to move these files into the .gitignor file. 

PS: Jak create the entire file structure for our team